presto-mysql/src/test/java/com/facebook/presto/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -204,6 +206,30 @@ public void testDecimalExceedingPrecisionMax()
         testUnsupportedDataType("decimal(50,0)");
     }
 
+    @Test
+    public void testDate()
+    {
+        LocalDate northernHemisphereWinter = LocalDate.of(2017, 1, 1);
+        LocalDate northernHemisphereSummer = LocalDate.of(2017, 7, 1);
+        // America/Sao_Paulo is interesting because it has DST changes at midnight. However, for this values to be useful, tests need to be run with JVM in that time zone.
```

Comment:
While Kathmandu is really cool zone, it doesn't have DST and never had....

Reply:
I saw the change to `pom.xml` in some other PR. Why isn't it here?